### DISCUSSION TITLE
Sonoff ZB Bridge how to set Zigbee channel manually

### DISCUSSION THREAD

**Discussion by dallibab** [arendst/Tasmota#10726] [Q&A] [answered]
I have my Sonoff ZB Bridge flashed and working with Tasmota, devices are working most of the time but I am getting some unresponsive issues periodically, From what I have read the Zigbee channel by default is set to 11.  I have neighbor's using that equivalent Wi-Fi channel.  How can I set the Zigbee channel manually ? I am fine re-flashing the Sonoff bridge again I just cant find this info anywhere on the web.

Thanks

**Comment by s-hadinger** [comment]
`ZbConfig` command

**Comment by dallibab** [comment]
The command doesn't show anything, I think this is because it's connected
to ha.

Can you point me to anyguides to setting this up from the start so I can
start from scratch.

Thanks for the reply.

**Reply by s-hadinger** [reply]
We don't support ZHA here. Please reach out to their own support channels.

**Comment by dallibab** [comment, marked as answer]
Sorry I thought this was the tasmota channel for sonoff zb bridge. Can you
point me in the right direction?,. I'm not fussed about the ha side of
things just want to configure the device to my preferred setting (manually
setting the ZigBee channel etc} I can try to work out the rest once it's
setup as per my requirements.

Thanks

**Reply by sfromis** [reply]
If you run ZHA, Tasmota is merely acting as a bridge, without any involvement in the details. The "primary" mode for Tasmota on the ZbBridge is that commands and statuses are communicated via MQTT. ZHA is a Home Assistant feature, which is where you should be able to get support.

**Comment by dallibab** [comment]
Thanks, I'll check it out.
